repository_templates/python/.github/workflows/python-release.yaml

```diff
@@ -0,0 +1,48 @@
+name: Python Release
+on:
+  pull_request_target:
+    types: [ closed ]
+    paths:
+      - 'python/grafana_foundation_sdk/**'
+
+env:
+  PYTHON_VERSION: '3.11'
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true && github.base_ref == '{{ .Extra.ReleaseBranch }}'
+
+    name: Build and release
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write  # mandatory for trusted publishing
+
+    environment:
+      name: pypi
+      url: https://pypi.org/p/grafana_foundation_sdk
+
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./python
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python {{ `${{ env.PYTHON_VERSION }}` }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: {{ `${{ env.PYTHON_VERSION }}` }}
```

Comment:
```suggestion
          python-version: "{{ env.PYTHON_VERSION }}"
```
This should work 🤔

Reply:
Nope: this file is a Go template, that's why I'm escaping this bit :) (it would otherwise fail since `env.PYTHON_VERSION` doesn't exist in the template's context)